client/X11/xf_utils.h

```diff
@@ -89,3 +89,5 @@ int LogTagAndXConvertSelection_ex(const char* tag, const char* file, const char*
 int LogDynAndXConvertSelection_ex(wLog* log, const char* file, const char* fkt, size_t line,
                                   Display* display, Atom selection, Atom target, Atom property,
                                   Window requestor, Time time);
+
+bool IsGnome();
```

Comment:
* `bool` is defined in `stdbool.h` (or use `BOOL` with `winpr/wtypes.h`)
* `fkt()` is `C` for arbitrary arguments, use `fkt(void)` instead

Reply:
Fixed. Once again, I apologize. I've never written in C for operating systems, only in my youth for controllers =) I used BOOL because it is used in most cases in the project.

Reply:
no worries, everyone has to start somewhere :grin: